features/admin/sign_in.feature

```diff
@@ -1,8 +1,8 @@
 @administrator_login
-Feature: Sign in to the store
-    In order to view my orders
-    As a Visitor
-    I want to be able to log in to the store
+Feature: Sign in to the administration
```

Comment:
Maybe "admin panel" instead.

Reply:
Changed to `administration panel`, because this name already appears in other tests